INSTALL.rst.txt

```diff
@@ -393,9 +251,3 @@ where INSTALLDIR is, for example, /usr/lib/python3.4/site-packages/.
      ldd /path/to/ext_module.so
 
    can be useful.
-
```

Comment:
The sentence "where INSTALLDIR is, for example" should be indented to render properly. Can you fix as part of this PR?